lib/loggerator/rails.rb

```diff
@@ -1,11 +1,50 @@
 require_relative "loggerator"
+require_relative "rails/rack/log_subscriber"
 
 module Loggerator
-  class Railtie < Rails::Railtie
+  module Rails
```

Comment:
@reidmix This, in conjunction with the Railtie inclusion of Loggerator in to the various Rails classes causes `Rails.env` to find `Loggerator::Rails`, instead of `::Rails`. To fix this, when testing with drydock, I had to change all references of `Rails.env` to `::Rails.env`. Don't think that's good. Given that it's never called directly, I'm considering making it `Loggerator::Railz` or something of the like to avoid that conflict.